web/packages/teleport/src/Integrations/EditAwsOidcIntegrationDialog.tsx

```diff
@@ -218,7 +218,7 @@ export function EditAwsOidcIntegrationDialog(props: Props) {
                     setConfirmed(e.target.checked);
                   }}
                 />
-                I have ran the command
+                I ran the command
```

Comment:
Suggestion:
```suggestion
                Next
```

Reply:
This is text for a checkbox. I'm not sure `[ ] Next` is the correct label for it.